terraform.tf.j2

```diff
@@ -24,6 +24,14 @@ locals {
   name        = "dataworks-aws-ucfs-claimant-consumer"
   environment = terraform.workspace == "default" ? "development" : terraform.workspace
 
+  env_prefix = {
+    development = ".dev"
+    qa          = ".qa"
+    integration = ".int"
+    preprod     = ".pre"
+    production  = ""
+  }
```

Comment:
I think these should go in the terraform bootstrap Secret Manager secret instead

Reply:
We will move these in the next story